.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -23,7 +23,7 @@ body:
         - This question is directly related to ccxt
         - You have read the [FAQ](https://github.com/ccxt/ccxt/wiki/FAQ) for most frequently asked questions
         - You have read the [How to submit a good issue](https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue)
-        - Make sure your local version of CCXT is up to date. Check by comparing the output of `ccxt.version` to https://github.com/ccxt/ccxt/blob/master/package.json#L3
+        - Make sure your local version of CCXT is up to date, buy comparing your `ccxt.version` output to [this version](https://github.com/ccxt/ccxt/blob/master/package.json#L3)
```

Comment:
typo "buy"

Reply:
uh oh :)) you know, you can even dont spend time commenting for it and directly push change from UI. i've edited